src/promptflow-evals/tests/unittests/generated_qa_chat_conv.jsonl

```diff
@@ -0,0 +1,5 @@
+{"messages":[{"role":"user","content":"What is Compute Instance?"},{"role":"assistant","content":"Compute instance is ..."}]}

```

Comment:
>> Put all test data/configs under src/promptflow/tests/test_configs.

Just read this from dev_setup.md.
https://github.com/microsoft/promptflow/blob/main/docs/dev/dev_setup.md